charts/nri-bundle/requirements.yaml

```diff
@@ -48,4 +48,4 @@ dependencies:
   - name: newrelic-infra-operator
     repository: file://../newrelic-infra-operator
     condition: newrelic-infra-operator.enabled
-    version: 0.3.0
```

Comment:
We usually do not bound together nri-bundle with its dependencies, in the future it is better to open different Prs 😄